indico/modules/events/editing/models/revisions.py

```diff
@@ -176,3 +181,14 @@ def is_submitter_revision(self):
     @property
     def is_editor_revision(self):
         return self.type.is_editor_action
+
+    @listens_for(orm.mapper, 'after_configured', once=True)
+    def _mappers_configured():
+        from indico.modules.events.editing.models.comments import EditingRevisionComment
+        query = (select([db.func.greatest(db.func.max(EditingRevisionComment.modified_dt),
+                                          db.func.max(EditingRevisionComment.created_dt),
+                                          EditingRevision.modified_dt, EditingRevision.created_dt)])
+                 .where(EditingRevisionComment.revision_id == EditingRevision.id)
+                 .correlate_except(EditingRevisionComment)
+                 .scalar_subquery())
+        EditingRevision.last_update_dt = column_property(query)
```

Comment:
Do we usually need it? If not I'd make it deferred and only `undefer()` it when needed.